### PR TITLE
CMR-6147 Added EMAIL_SUBSCRIPTION_MANAGEMENT acl for MMT

### PR DESCRIPTION
**Pull request by sxu123**

### CODE REVIEW

File: access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj [hunk 1]
```diff
@@ -1287,6 +1287,52 @@
...
+                                              :target "EMAIL_SUBSCRIPTION_MANAGEMENT"}}
+        acl (access-control/create-acl (test-util/conn-context)
+                                       subscription-acl 
+                                       {:token token})]
+    (is (= subscription-acl
+           (access-control/get-acl (test-util/conn-context)
+                                   (:concept_id acl)
+                                   {:token token})))
+
+    (testing "MMT SUBACRIPTION Acl permissions"
```
Comment (charleshuang80) [inline access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj]: is this supposed to be 'SUBSCRIPTION'?

Comment (sxu123) [reply access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj]: Good catch. Done.

File: access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj [hunk 2]
```diff
@@ -1287,6 +1287,52 @@
               (get-in (access-control/get-acl (test-util/conn-context) (:concept_id acl4) {:token token})
                       [:catalog_item_identity :collection_identifier])))))))
 
+(deftest CMR-6147-subscription-mmt-acl-test
```
[the remark this answers is not in the document]
Comment (daniel-zamora) [reply access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj]: And other references to MMT

[the remark this answers is not in the document]
Comment (sxu123) [reply access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj]: Changed it to email-subscription-acl-test

**Pull request merged**